### ISSUE TITLE
Validation set

### ISSUE THREAD

**Issue by Arthur023** [RockyXu66/Faster_RCNN_for_Open_Images_Dataset_Keras#51]
Hey guys I almost got the code working. But now I'm wondering where you can put in the validation set. Or does the code just takes some random samples from the training set and makes them the validation set?

Thanks

**Comment by Arthur023** [comment]
Is there any chance that someone can help me out here. I have been looking for two days but can't find any place for a validation set. 
If someone has some more information about training_on_batch that would help as well. I'm not finding any good documentation about it

**Comment by AlOo2002** [comment]
did you find it

**Comment by charinho8** [comment]
> Is there any chance that someone can help me out here. I have been looking for two days but can't find any place for a validation set.
> If someone has some more information about training_on_batch that would help as well. I'm not finding any good documentation about it

did you have any luck solving this problem?

**Comment by Gerro385** [comment]
It is a bit annoying but you can do like 1000 epochs of training, save the record then open the test notebook and do some testing. Afterwards take the maP and but it in the record.csv.
Do this for each round of Training.
